xroad-catalog-collector/README.md

```diff
@@ -29,90 +29,120 @@ X-Road Catalog Collector can be built by running:
+
+#### Mandatory configurations required by common features
+
+| Configurations                            | Required | Defaults                                     | Description                                                                                                                                                                                           | Since |
+|-------------------------------------------|----------|----------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------|
+| `xroad-catalog.target.xroad-instance`     | Y        |                                              | A parameter for setting the X-Road instance.                                                                                                                                                          | 1.0.0 |
+| `xroad-catalog.target.subsystem-code`     | Y        |                                              | A parameter for setting the X-Road sub-system code.                                                                                                                                                   | 1.0.0 |
+| `xroad-catalog.target.member-class`       | Y        |                                              | A parameter for setting the X-Road member class.                                                                                                                                                      | 1.0.0 |
+| `xroad-catalog.target.member-code`        | Y        |                                              | A parameter for setting the X-Road member code.                                                                                                                                                       | 1.0.0 |
+| `xroad-catalog.urls.security-server-host` | Y        |                                              | A parameter for setting the security server host to connect to. e.g., http://security.server.host:8080                                                                                                | 1.0.0 |
+| `xroad-catalog.urls.list-clients-host`    | Y        | `${xroad-catalog.urls.security-server-host}` | A parameter for setting the security server host to be used to get list of clients. Default value references `xroad-catalog.urls.security-server-host`.                                               | 1.0.0 |
+| `xroad-catalog.urls.webservices-endpoint` | Y        | `${xroad-catalog.urls.security-server-host}` | A parameter for setting the security server host to be used to get list of OpenAPI services, SOAP services, and list of services. Default value references `xroad-catalog.urls.security-server-host`. | 1.0.0 |
```

Comment:
I think these aren't actually mandatory, since they default to the `xroad-catalog.urls.security-server-host`.